ui/lib/pki/addon/controllers/issuers/index.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+
+export default class PkiRolesIssuerController extends Controller {
+  // To prevent production build bug of passing D.actions to on "click": https://github.com/hashicorp/vault/pull/16983
+  @action onLinkClick(D) {
+    D.actions.close();
```

Comment:
This needs to be wrapped in the `next` method from `@ember/runloop`:
```js
next(() => D.actions.close());
```

Reply:
Ah... Okay. I see that now on the PR.